source/Camera/Basic/CaptureHDRCompleteSettings/CaptureHDRCompleteSettings.cs

```diff
@@ -0,0 +1,67 @@
+/*
+This example shows how to acquire an HDR image from the Zivid camera with fully
+configured settings for each frame. In general, taking an HDR image is a lot
+simpler than this as the default settings work for most scenes. The purpose of
+this example is to demonstrate how to configure all the settings.
+*/
+
+using System;
+using System.Collections.Generic;
+using Duration = Zivid.NET.Duration;
+
+class Program
+{
+    static void Main()
+    {
+        try
+        {
+            var zivid = new Zivid.NET.Application();
+
+            Console.WriteLine("Connecting to the camera");
+            var camera = zivid.ConnectCamera();
+
+            Console.WriteLine("Configuring settings same for all HDR frames");
+
+            var settingsDefault = new Zivid.NET.Settings
+            {
+                Brightness = 1,
+                Bidirectional = false,
+                BlueBalance = 1.081,
+                RedBalance = 1.709
```

Comment:
You should be able to do 
                Filters = {
                    Gaussian = {
                        Enabled = true,
                        Sigma = 2.245,
                    }
                },

and so on inside here, and drop the stuff below

Reply:
done